doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-designer-test-scenario-import-spreadsheet-proc.adoc

```diff
@@ -0,0 +1,16 @@
+[id='test-designer-test-scenario-import-spreadsheet-proc']
+= Importing a test scenario spreadsheet
+
+Follow the procedure below to import a test scenarios spreadsheet data using the Test Scenario designer.
```

Comment:
Are we exporting a test scenario spreadsheet or the test scenario's spreadsheet data?

s/a test scenarios spreadsheet data/a test scenario's spreadsheet data
or
s/a test scenarios spreadsheet data/a test scenario spreadsheet

If it is only the data that can be imported and exported, then perhaps the wording should differentiate between the test scenario spreadsheet in the Test Scenario designer and the data file?

Reply:
Done.

Reply:
Again, "local directory" not file system.